tests/drivers/dac/dac_loopback/src/test_dac.c

```diff
@@ -44,6 +44,36 @@
 #define ADC_REFERENCE		ADC_REF_INTERNAL
 #define ADC_ACQUISITION_TIME	ADC_ACQ_TIME_DEFAULT
 
+#elif defined(CONFIG_BOARD_TWR_KE18F)
+
+/* DAC0 output on PTE9, ADC input read from ADC0_SE12 */
```

Comment:
@MaureenHelm As mentioned previously, according to the KE1xF reference manual, I should be able to read back the DAC output voltage on ADC0_SE23 (similar to what works for the K6x, below). This does not seem work for the KE1xF, though. Routing to an external ADC input (here, ADC0_SE12) works.

Reply:
Yep, I'll follow up again internally with the hardware team on this